build.sh

```diff
@@ -103,6 +103,22 @@ for i in "${ARGS[@]}"; do
 	esac;
 done
 
+if ! command -v /bin/bear &> /dev/null; then
+	echo "'bear' executable not found. Compilation database will not be built"
+elif [ -z "${DO_NOT_EXEC_BEAR+x}" ]; then
+	b_log "Running bear on top of build script"
+
+	OUTPUT_FILE=_build/"$TARGET"/compile_commands.json
```

Comment:
maybe put full value in `"`
```suggestion
	OUTPUT_FILE="_build/$TARGET/compile_commands.json"
```